com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md

```diff
@@ -5,7 +5,7 @@ This table provides an overview of the current features supported in the Univers
 **Note:** If a feature is marked __In research__, the URP team is still researching how and when to implement the feature. If a feature is marked as __Not supported__, it's because Unity is not planning to support it in any release. 
 
 
-| Feature                                                      | Built-in Render Pipeline<br/>Unity 2019.3                               | Universal Render Pipeline                                    |
+| Feature                                                      | Built-in Render Pipeline<br/>Unity 2019.x                               | Universal Render Pipeline                                    |
```

Comment:
2020.x?

Reply:
If there's no difference between 2019.x and 2020.x, should we mention both?
"Built-in Render Pipeline<br/>Unity 2019.x, Unity 2020.x."